### DISCUSSION TITLE
how can i make the fnf neo mod 3.0 main menu?

### DISCUSSION THREAD

**Discussion by ErenKaya31** [DEMOLITIONDON96/Demolition-Engine#55] [Source Q&A] [answered]
i need it for a upgrade tho

**Comment by RealDealJinx** [comment, marked as answer]
mess with the X and Y ig

**Reply by ErenKaya31** [reply]
bruh dont mark answers

**Reply by ErenKaya31** [reply]
i need the code like transitions

**Reply by ThatOneSillyGuyDon** [reply]
> i need the code like transitions

Bro, do the code yourself, we’re not gonna make it and do the work for you, lmfao, I’m marking this as an answer because Matt already gave you your answer

**Comment by ghost** [comment]
If.
You.
Want.
To.
Copy.
Code.
From.
Other.
Mods.
Then.
Find.
Their.
Source.
Code.
On.
Github.
